alvr/server_openvr/src/lib.rs

```diff
@@ -392,16 +397,13 @@ extern "C" fn wait_for_vsync() {
             .read()
             .as_ref()
             .and_then(|ctx| ctx.duration_until_next_vsync())
+            .unwrap_or(PRE_HEADSET_STATS_WAIT_INTERVAL)
     } else {
-        None
+        // Fallback to avoid deadlocking people's systems accidentally
+        Duration::from_micros(1000)
```

Comment:
Maybe just use `from_millis(1)`?

Reply:
Honestly I was debating dropping it to 500, but sure